src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/Resource.sql

```diff
@@ -1,4 +1,22 @@
-﻿CREATE TABLE dbo.Resource
+﻿CREATE TABLE dbo.CurrentResource -- This is replaced by view CurrentResource
```

Comment:
qq: why is this created and dropped straight away as a table?

Reply:
I put elaborate comment in the code.